<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpscaleTaskDialog</class>
 <widget class="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upscale Task</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Multiplier:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="multiplierLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string notr="true">2</string>
        </property>
        <property name="alignment">
         <enum>Qt::AlignCenter</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Listfiles File:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWidget" name="listfilesFileLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="listfilesFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Skip step:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListWidget" name="skipStepListWidget">
        <property name="selectionMode">
         <set>QAbstractItemView::MultiSelection</set>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Regular CEL Files</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object CEL Files</string>
         </property>
        </item>
        <!--item>
         <property name="text">
          <string>Special CEL Files</string>
         </property>
        </item-->
        <item>
         <property name="text">
          <string>Cutscenes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Art CEL Files</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CL2 Files of Missiles</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CL2 Files of Monsters</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CL2 Files of PlrGFX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed CL2 Files</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tilesets</string>
         </property>
        </item>
        <!--item>
         <property name="text">
          <string>Fixed Tilesets</string>
         </property>
        </item-->
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Patch Tilesets:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="patchTilesetsCheckBox">
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Fix and optimize the original tilesets</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Assets Folder:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="LineEditWidget" name="assetsFolderLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="assetsFolderBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Output Folder:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="LineEditWidget" name="outputFolderLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="outputFolderBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upscaleButton">
       <property name="text">
        <string>Upscale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upscaleCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
